phac-nml/irida-next
# Internal: Dockerfile

This PR adds in a Dockerfile with the aim of using this for production deployment.

Fixes #18 

To test this run the following commands:
1. `docker build -t irida-next .`
2. `docker run --network="host" -e DATABASE_URL="postgres://test:test@localhost/irida_next_prod" -e SECRET_KEY_BASE="SUPERSECRETKEY" irida-next:latest bin/rails db:setup`
3. `docker run --network="host" -e DATABASE_URL="postgres://test:test@localhost/irida_next_prod" -e SECRET_KEY_BASE="bopbloblaw" -e RAILS_SERVE_STATIC_FILES=true -p 3000:3000 irida-next:latest`

After running all those commands you should have a running dockerized build or irida-next available at http://localhost:3000
